pydarn/exceptions/plot_exceptions.py

```diff
@@ -175,3 +179,16 @@ def __init__(self, parameter: str, gate_num: int, max_range_gate: int):
                                              max_gate=self.max_range_gate-1)
         super().__init__(self.message)
         pydarn_log.error(self.message)
+
+
+class NoChannelError(Exception):
+    """
+    This error is raised when the channel specified by user
+    does not exist
+    """
+    def __init__(self, channel: int):
+        self.channel = channel
+        self.message = "There is no data for channel {channel}"\
+                       .format(channel=self.channel)
```

Comment:
Similar to my no data warning maybe we could suggest a channel that data has. This would help with say "twofsound" when we have 1 and 2 and when we have stero mode with 0 and 1

Reply:
I've added the option to give an option in the exception, but the implementation of the exception in fan.py is a bit rudimentary at the moment, assuming I will probably change how it works when I sort out the other issue below. 


(I also know why it's doing that other issue, but need to have a think about how to go around it so it preserves the fix for the issue with the HAN data too)